tools/generate_conda_file.py

```diff
@@ -56,7 +56,7 @@
     "scikit-surprise": "scikit-surprise>=1.0.6",
     "swig": "swig==3.0.12",
     "lightgbm": "lightgbm==2.2.1",
-    "cornac": "cornac>=1.1.2",
```

Comment:
please update setup.py section as well in the new changes to staging, we're probably going to start deprecating conda

Reply:
fixed!